2024/half-2/day-12/array-at/array-at.test.ts

```
@@ -0,0 +1,77 @@
+    });
+
+    test("out-of-range", () => {
+      expect(arr.myAt(3)).toBeUndefined();
+      expect(arr.myAt(4)).toBeUndefined();
+      expect(arr.myAt(-4)).toBeUndefined();
+      expect(arr.myAt(-5)).toBeUndefined();
+    });
+  });
+
+  test("sparse arrays", () => {
+    const arr = [1, 2, , 4];
```

Biome

[error] This array contains an empty slot.

Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)